### ISSUE TITLE
Alpha Fold Colab Pro not working

### ISSUE THREAD

**Issue by LuikartLab** [deepmind/alphafold#146]
I have been using the colab notebook version with success to generate several structures thus far.  Today It has stopped working.  I am using colab pro with a high ram runtime and have done a factory reset all.  Everthing seems good until I search against genetic databases.  The following is the error I get - 

NotFoundError                             Traceback (most recent call last)
<ipython-input-14-1640ae7ce3a3> in <module>()
     24 import py3Dmol
     25 
---> 26 from alphafold.model import model
     27 from alphafold.model import config
     28 from alphafold.model import data

6 frames
/usr/local/lib/python3.7/dist-packages/tensorflow/python/framework/load_library.py in load_library(library_location)
    152 
    153     for lib in kernel_libraries:
--> 154       py_tf.TF_LoadLibrary(lib)
    155 
    156   else:

NotFoundError: /opt/conda/lib/python3.7/site-packages/tensorflow/core/kernels/libtfkernel_sobol_op.so: undefined symbol: _ZN10tensorflow6StatusC1ENS_5error4CodeEN4absl14lts_2020_09_2311string_viewEOSt6vectorINS_10StackFrameESaIS7_EE

**Comment by LuikartLab** [comment]
ISSUE SOLVE - 
Hi, I used the last version of Alphafold notebook however I got the same error. Also, I'm sure that I run all cells in order. I tried to install Tensorflow 2.2.0 but I got another error in the Alphafold Prediction step. When I install Tensorflow 2.5.0 by !pip install -U tensorflow==2.5.0 at the begging of the notebook all problems were solved and notebook run properly.

I did the above and it appears to be running again.  Sorry, I am guilty of writing up an issue without looking over recent issues and their fixes.
